drivers/ethernet/eth_sam_gmac.c

```diff
@@ -1792,9 +1792,14 @@ static void get_mac_addr_from_i2c_eeprom(uint8_t mac_addr[6])
 		return;
```

Comment:
Almost forgot, please change the commit subject to `drivers: ethernet: gmac: Fix coverity issue`

Reply:
Fixed 😄